storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5688,6 +5688,30 @@ static int rocksdb_init_internal(void *const p) {
   }
 
   if (rocksdb_persistent_cache_size_mb > 0) {
+    // TODO: This is the limitation in RocksDB.
```

Comment:
Please replace this comment with the explanation where `cache_file_size` is set to 100MB and where is the validation of the value provided with `rocksdb_persistent_cache_size_mb`. That will explain why we need to do the validation prior to calling `NewPersistentCache`